packages/core/src/lib/list-box.svelte

```diff
@@ -0,0 +1,249 @@
+<!--
+    @component
+    Two lists of elements that can be moved between the two lists
+
+    ```svelete
+     <ListBox
+        disabled = {false}
+        left ='leftText1,leftText2,leftText3',
+        right = 'rightText1,rightText2,rightText3'
+        leftlabel = 'Left'
+        rightlabel = 'Right'
+        height = '200px'
+        suffix= {false}
+        />
+    ```
+    
+-->
+<svelte:options immutable />
+
+<script lang="ts">
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
To my knowledge, this isn't required elsewhere, why is it required here? Also, prefer per-line ignores to per-file ignores